src/components/Filters/LabelFilter.tsx

```diff
@@ -32,21 +32,21 @@ export class LabelFilters extends React.Component<LabelFiltersProps> {
         />
         <Popover
           headerContent={<span>Label Filter Help</span>}
+          position={PopoverPosition.right}
           bodyContent={
             <>
               To set a label filter you must enter values like.
               <br />
               <ul style={{ listStyleType: 'circle', marginLeft: '20px' }}>
                 <li>Filter by label presence: label</li>
-                <li>Filter by label and value: label=value</li>
+                <li>Filter by label and value: label:value</li>
                 <li>
-                  Filter by more than one label and one or more values: label=value label2=value2,value2-2 (separate
+                  Filter by more than one label and one or more values: label:value label2:value2,value2-2 (separate
```

Comment:
I suggest to add a couple of <br />
```
                <li>
                  Filter by more than one label and one or more values:
                  <br />
                  label:value label2:value2,value2-2
                  <br />
                  (separate with ' ')
                </li>
```
![image](https://user-images.githubusercontent.com/1662329/78236383-35a63900-74da-11ea-82c2-9370b87ac56f.png)

In that way the legend is not broken and I think it can be read better.

It's minor